pkg/api-server/inspect_endpoints.go

```diff
@@ -55,6 +74,29 @@ func addInspectEndpoints(
 			Returns(200, "OK", nil),
 	)
 
+	if cfg.Mode != config_core.Global {
+		ws.Route(
+			ws.GET("/meshes/{mesh}/dataplanes/{dataplane}/xds").To(inspectDataplaneXDS(cd, rm, cfg.GetEnvoyAdminPort())).
+				Doc("inspect dataplane XDS configuration").
+				Param(ws.PathParameter("mesh", "mesh name").DataType("string")).
+				Param(ws.PathParameter("dataplane", "dataplane name").DataType("string")),
+		)
+
+		ws.Route(
+			ws.GET("/zoneingresses/{zoneingress}/xds").To(inspectZoneIngressXDS(cd, rm, cfg.Multizone.Zone.Name, cfg.GetEnvoyAdminPort())).
+				Doc("inspect zone ingresses XDS configuration").
+				Param(ws.PathParameter("zoneingress", "zoneingress name").DataType("string")),
+		)
+	} else {
```

Comment:
That's unfortunate. This makes this feature really hard to use with multizone deployment. Can we figure out a solution so this also works with multizone?

Reply:
I think we don't have an address of the Zone CP on Global CP, KDS works both ways but it reuses a connection opened by Zone CP. This makes it kind of impossible today to simply forward API requests to the corresponding Zone CP. Probably we can try to reach out to `/config_dump` through the ZoneIngress, so the path will look like: Global CP -> Zone Ingress -> DPP, but I'm not sure it's a good solution. 
For this release, It's still will be possible to get config dumps using `kumactl`, so I think it's not that bad for multizone.

Reply:
Yep sounds ok to have this shortcoming in this release. Can you open an issue to add support for this?

Reply:
https://github.com/kumahq/kuma/issues/3789